README.md

```diff
@@ -8,24 +8,65 @@ ospool:
 In this lesson, we'll learn how to troubleshoot jobs that never start or fail in unexpected ways. 
 
 # Troubleshooting techniques
+We will look into 4 criterias. The criteria's are listed below
 
-## Diagnostics with condor_q
+ - [Job is held](#job-is-held)
+ - [Job completed but was unsuccessful](#job-completed-but-was-unsuccessful)
+ - [Job does not start](#Job-does-not-start)
+ - [Job is running longer than expected](#job-is-running-longer-than-expected)
 
-The `condor_q` command shows the status of the jobs and it can be used 
-to diagnose why jobs are not running. Using the `-better-analyze` flag 
-with `condor_q` can show you detailed information about why a job isn't 
-starting on a specific pool. Since OSG Connect sends jobs to many places, we also need to 
-specify a pool name with the `-pool` flag.                              
+## Job is held
+HTCondor puts your jobs on hold when something goes wrong in the process of managing your jobs. HTCondor provides a ``Hold Reason`` that explains what went wrong. To see what went wrong you can use the command `condor_q -hold`. A typical hold message may look like the following
 
-Unless you know a specific pool you would like to query, checking the `flock.opensciencegrid.org` pool is usually a good place to start.
+	$ condor_q -hold
+	ap40.uw.osg-htc.org : <128.105.68.62:9618?... @ 08/01/24 15:01:50
+ 	ID         OWNER          HELD_SINCE  HOLD_REASON
+	130.0       alice           8/1  14:56 Transfer input files failure at ⋯
 
-	$ condor_q -better-analyze JOB-ID -pool POOL-NAME
+In this particular case, a user had this in his or her HTCondor submit file:
+
+	transfer_output_files = outputfile
+
+However, when the job executed, it went into Held state. To see more about the error message `condor_q -hold JOB-ID` can be used. An example error message in this scenario is shown below:
+
+	$ condor_q -hold JOB-ID
+
+	Transfer output files failure at access point… while receiving files from the execution point. Details: Error from ….execute point … failed to send file(s) at apxx; failed to read from file /path: (errno 2) No such file or directory
+
+The reason why the file transfer failed is because `outputfile` was never created on the worker node. Remember that at the beginning we said that the user specifically requested `transfer_outputfiles = outputfile`! HTCondor could not complete this request, and so the job went into Held state instead of finishing normally.
+
+It's quite possible that the error was simply transient, and if we retry, the job will succeed. We can re-queue a job that is in Held state by using `condor_release`: 
+
+	condor_release JOB-ID 
```

Comment:
Do we want to include other common hold reasons / examples?

Reply:
Done